vigetlabs/wordpress-site-starter
# [#13] Alert Banner UI

# Summary

This PR adds UI for the Alert Banner component

## Issues

- #13 

## Testing Instructions

1. Add an Alert Banner block to your page
2. Add paragraph text, and optionally a button. Also optionally, make the block "dismissible"
3. Check alignment of paragraph, button, and "dismiss" button and compare to design screenshots in parent issue


Dismissible Block w/Button:

<img width="1440" alt="Screenshot 2024-04-25 at 4 57 32 PM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/2685062/df215c88-9a53-42a4-ba9b-88cd68733623">

<img width="497" alt="Screenshot 2024-04-25 at 4 57 47 PM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/2685062/5d98a856-1ff4-4daf-8aa7-df0a8cb46c93">

Non-dismissible Block w/Button:

<img width="497" alt="Screenshot 2024-04-25 at 4 58 22 PM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/2685062/3a9ad279-fa0e-47bc-b35e-6df4fdd36244">

<img width="1440" alt="Screenshot 2024-04-25 at 4 58 33 PM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/2685062/7deb4c7f-01fd-4bac-a5ee-0a27aaa09a1d">
